umep/functions/SOLWEIGpython/WriteMetadataSOLWEIG.py

```diff
@@ -1,166 +1,210 @@
 from builtins import str
+
 # This file prints out run information used for each specific run
 from time import strftime
-from osgeo import osr, gdal
-import sys
-import qgis.core
 
```

Comment:
@biglimp I've removed these imports from this file because they trigger `osgeo` and `qgis` dependencies which make installation of `umep` more complicated as a standalone Python package (when used outside of QGIS). I propose that we use the transform and CRS info directly per line 81 below.